This patch is taken from upstream.  It fixes a LaTeX test that is broken due
to using a Skylighting version greater than or equal to 0.8.1.1.

From 2cd1c7b30f200d18f1f1bdef1671369e1ad303ed Mon Sep 17 00:00:00 2001
From: John MacFarlane <jgm@berkeley.edu>
Date: Thu, 13 Jun 2019 19:33:52 -0700
Subject: [PATCH] Update test for skylighting 0.8.1.1.

---
 test/lhs-test.latex | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/test/lhs-test.latex b/test/lhs-test.latex
index 0442db5ca..a456995a3 100644
--- a/test/lhs-test.latex
+++ b/test/lhs-test.latex
@@ -94,9 +94,9 @@ return a single value:
 
 \begin{Shaded}
 \begin{Highlighting}[]
-\OtherTok{unsplit ::}\NormalTok{ (}\DataTypeTok{Arrow}\NormalTok{ a) }\OtherTok{=>}\NormalTok{ (b }\OtherTok{->}\NormalTok{ c }\OtherTok{->}\NormalTok{ d) }\OtherTok{->}\NormalTok{ a (b, c) d}
+\OtherTok{unsplit ::}\NormalTok{ (}\DataTypeTok{Arrow}\NormalTok{ a) }\OtherTok{=>}\NormalTok{ (b }\OtherTok{{-}>}\NormalTok{ c }\OtherTok{{-}>}\NormalTok{ d) }\OtherTok{{-}>}\NormalTok{ a (b, c) d}
 \NormalTok{unsplit }\OtherTok{=}\NormalTok{ arr }\OperatorTok{.} \FunctionTok{uncurry}
-          \CommentTok{-- arr (\textbackslash{}op (x,y) -> x `op` y)}
+          \CommentTok{{-}{-} arr (\textbackslash{}op (x,y) {-}> x \textasciigrave{}op\textasciigrave{} y)}
 \end{Highlighting}
 \end{Shaded}
 
-- 
2.23.0

